independent-projects/arc/pom.xml

```diff
@@ -88,9 +88,16 @@
             </dependency>
 
             <dependency>
-                <groupId>junit</groupId>
-                <artifactId>junit</artifactId>
-                <version>${version.junit4}</version>
+                <groupId>org.junit.jupiter</groupId>
+                <artifactId>junit-jupiter-api</artifactId>
```

Comment:
You can depend on `org.junit.jupiter:junit-jupiter` only (which will bring the necessary dependencies)

Reply:
Good point, done that!